CODEOWNERS

```
@@ -1,2 +1,2 @@
 # code owners for amplitude-dev-center repo
-*	brian.giori@amplitude.com justin.fiedler@amplitude.com
+*	brian.giori@amplitude.com justin.fiedler@amplitude.com  mark.zegarelli@amplitude.com
```

nit: remove extra space?
```suggestion
*	brian.giori@amplitude.com justin.fiedler@amplitude.com mark.zegarelli@amplitude.com
```